packages/workbox-core/test/bundle/node/utils/test-logHelper.js

```diff
@@ -0,0 +1,339 @@
+import chai from 'chai';
+import sinon from 'sinon';
+
+import logHelper from '../../../../src/utils/logHelper';
+
+const expect = chai.expect;
+
+describe(`logHelper [${process.env.NODE_ENV}]`, function() {
+  let stubs = [];
+
+  beforeEach(function() {
+    logHelper.logLevel = logHelper.LOG_LEVELS.verbose;
+  });
+
+  afterEach(function() {
```

Comment:
Good idea - will switch over now.

Do you think we should add some custom "project lint" checks (i.e. regexs for sinon.(spy|stub) to somewhat enforce these kinds of rules?)

I'm largely thinking on this as an interested considering that Phil's PR's have a lot of consistency stuff being done and it's a shame it's not enforced in any way.